### ISSUE TITLE
Julian Calendar can not calculate day of year.

### ISSUE THREAD

**Issue by ledm** [Unidata/cftime#70]
There's an error in the num2date function when using the Julian calendar in `cftime`. The `num2date` does not correctly produce the day of year:
ie:

```
import cftime
In [6]: cftime.num2date(16,'day since 1950-01-01 00:00:00.0000000',calendar='365_day')
Out[6]: cftime._cftime.DatetimeNoLeap(1950, 1, 17, 0, 0, 0, 0, 1, 17)

In [7]: cftime.num2date(16,'day since 1950-01-01 00:00:00.0000000',calendar='julian')
Out[7]: cftime._cftime.DatetimeJulian(1950, 1, 17, 0, 0, 0, 0, -1, 1)
```
The day of year (final number in the date output) is calculated correctly in the `365_day` calendar, but is set to `1` in the julian calendar.

**Comment by jswhit** [comment]
This was fixed in the 1.0.1 release

**Comment by jswhit** [comment]
Sorry, the fix was not in 1.0.1 - but it is fixed in master and that fix will be in the next release.
